configure.in

```diff
@@ -7263,6 +7260,10 @@ if test "$USE_WIN32" = 1 ; then
         fi
     fi
 
+    if test "$wxUSE_DIB" = "yes"; then
+        AC_DEFINE(wxUSE_WXDIB)
```

Comment:
Isn't `wxDIB` really wxMSW-specific? I.e. are we really going to use it in wxGTK code now? AFAICS so far we always use it in wxMSW code only and we have checks like `#if !defined(__WXMSW__) || wxUSE_WXDIB` in `src/html/m_image.cpp`.